Components/ThirdPlugins/SwagBundleHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SwagTax\Components\ThirdPlugins;
+
+use Doctrine\DBAL\Connection;
+use SwagTax\Structs\TaxMapping;
+use SwagTax\Structs\UpdateConfig;
+
+class SwagBundleHandler extends AbstractThirdPlugin
```

Comment:
shouldn't the plugin related classes be located in the referring plugin?

Reply:
Alles in allem hast du recht aber ich denke für diesen fall wäre es ein schritt zu weit. Sollten noch andere Plugins dazu kommen können wir das immer noch machen

Reply:
okay :+1: